doc/developing.rst

```diff
@@ -126,7 +122,114 @@ Building the HTML documentation
 
 To build the HTML documentation (output in ``doc/build/html``), run::
 
-  $ bin/sphinxbuilder
+  $ sphinx-build doc doc/build/html
+
+Developing Reg, Dectate or Importscan
+-------------------------------------
+
+If you need to adjust the sources of reg, dectate or importscan and test them
+together with morepath, they're available in the src directory.
+There you can edit them and test the changes with morepath, which uses them as
+dependencies when you install Morepath for developement as described above.
+
+Installing tox locally with with different Python versions
+----------------------------------------------------------
+
+If you want to check if Morepath works works with the supported Python versions
+and check if it pathes all Travis Ci tests, you can install tox locally.
+
```

Comment:
pathes -> passes.

I don't understand what you mean by "install tox locally". Oh, I think you mean instead of doing the continuous integration? I think a developer almost always needs to be able to run things with tox locally, so we should just call this "installing tox" and forget about the word "locally".